tests/scripts/deploy-validate-vault.sh

```diff
@@ -132,7 +132,10 @@ function validate_rgw_token {
   RGW_TOKEN_FILE=$(kubectl -n rook-ceph describe pods "$RGW_POD" | grep  "rgw-crypt-vault-token-file" | cut -f2- -d=)
   VAULT_PATH_PREFIX=$(kubectl -n rook-ceph describe pods "$RGW_POD" | grep  "rgw-crypt-vault-prefix" | cut -f2- -d=)
   VAULT_TOKEN=$(kubectl -n rook-ceph exec $RGW_POD -- cat $RGW_TOKEN_FILE)
-  
+  #VAULT_CACERT=$(kubectl -n rook-ceph describe pods "$RGW_POD" | grep  "rgw-crypt-vault-ssl-cacert" | cut -f2- -d=)
```

Comment:
Are these test commands still WIP?

Reply:
@travisn The required changes will be available on ceph v16.2.6, which is not yet released. Or else we need to add check in the test script to see whether it is ceph version is higher or not

Reply:
@thotz can you try with `ceph/daemon-base:latest-pacific-devel` to validate your patch?